### DISCUSSION TITLE
Using Badges as a "Verified" icon beside usernames

### DISCUSSION THREAD

**Discussion by quantnet** [VersoBit/XF-Badges#34] [Ideas]
This starts to look pretty nice and features are growing.
I have a Verified account badge that I manually assign to members I know personally who use their real name. Sort of like verified Twitter, FB account.
I wonder if we can designate a badge featured which will be show next to their username. So, my main use will be to have a small green check badge next to their username. Styling for this would be a bit different, smaller compared to the other badges.

**Comment by RyleaStark** [comment]
An addon already exists for this functionality, I fear my time would be better spent bringing other features into the addon before spending my time on this. 

https://xenforo.com/community/resources/xp-verification-badge.7285/

However I wont shut the idea down fully, rather as mentioned in the XF thread, Let me know your thoughts around what you're trying to do compared to what the above addon offers.
